src/js/pages/services/DeploymentsTab.js

```diff
@@ -152,10 +152,13 @@ class DeploymentsTab extends mixin(StoreMixin) {
   renderStatus(prop, deployment) {
     const title = `Step ${deployment.getCurrentStep()} of ${deployment.getTotalSteps()}`;
     const services = deployment.getAffectedServices();
+    const action = deployment.currentActions.map(function (action) {
+      return action.action;
```

Comment:
Shouldn't we ensure the action is affecting the current service? Keep in mind this will be on a per-app row. 

I wonder if this should be inside of the `map` starting at L157.